code/modules/admin/admin.dm

```diff
@@ -891,7 +892,6 @@ var/global/floorIsLava = 0
 			<A href='?src=\ref[src];secretsfun=maint_access_engiebrig'>Change all maintenance doors to engie/brig access only</A><BR>
 			<A href='?src=\ref[src];secretsfun=maint_access_brig'>Change all maintenance doors to brig access only</A><BR>
 			<A href='?src=\ref[src];secretsfun=infinite_sec'>Remove cap on security officers</A><BR>
-			<a href='?src=\ref[src];secretsfun=virus_custom'>Custom Virus Outbreak</a><BR>
```

Comment:
hey, I use this

Reply:
Don't worry, I'm replacing it with something infinitely better

Reply:
(see datumvars.dm)

Reply:
Oh but reguarding global outbreak like those, they're getting redone from scratch in part 3